probatus/utils/shap_helpers.py

```diff
@@ -178,12 +181,14 @@ def calculate_shap_importance(shap_values, columns, output_columns_suffix=""):
     # Find average shap importance for neg and pos class
     shap_abs_mean = np.mean(np.abs(shap_values), axis=0)
     shap_mean = np.mean(shap_values, axis=0)
+    penalized_shap_abs_mean = np.mean(np.abs(shap_values), axis=0) - np.std(np.abs(shap_values), axis=0) / 2.0
```

Comment:
Maybe the penalizing factor (2.0) should be a parameter as well (Default 1 which means its the same as it was before, 2 recommended, ...). What do you think?

Reply:
For the power user, there is value in being able to control this. Intuitively, we could think of it as the `std penalty factor`. I'd formulate as: `penalized_shap_abs_mean = (mean_shap - (std_shap * std_penalty_factor))`

As such, std_penalty_factor = 0, would be no penalty (disabled). std_penalty_factor = 0.5 would be as I have implemented above.

From an API usability POV, what is the preferred way to implement? We could:
1. Have two params: shap_variance_penalty (bool) & shap_variance_penalty_factor (I don't like this approach)
2. Have one param: shap_variance_penalty_factor

For 2, it could be disabled by either setting 
- shap_variance_penalty_factor = None, or
- shap_variance_penalty_factor = 0

Thanks for the feedback.

Reply:
I agree, I think approach 2 would make the most sense. My preference would be to keep it numerical but intuitive (factor=0). However, in the code we often make use of 'None' as default (https://github.com/ing-bank/probatus/blob/main/probatus/feature_elimination/feature_elimination.py), so to be more in line with the rest of the library I think using 'None' makes more sense. Please let me know your thoughts :)

Reply:
Yeah, that makes sense. I'll draft up something along those lines and test the code a bit more. Thanks